packages/loader/container-loader/src/container.ts

```diff
@@ -1508,9 +1512,15 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
                 this._audience.addMember(priorClient.clientId, priorClient.client);
             }
 
+            const deltaManagerForCatchingUp =
+                this.mc.config.getBoolean("Fluid.Container.CatchUpBeforeDeclaringConnected") === true ?
```

Comment:
Have you given a thought that it should be opt out and not opt in?

Reply:
Only a little.  I think shipping dark is the standard operating procedure, and there's no rush to diverge from this.  Also I personally want to practice what it's like to use feature gates.